2020/10/2020-10-23-RIAA.md

```diff
@@ -4,115 +4,27 @@ GitHub
 
 Dear Sir or Madam:
 
-I am contacting you on behalf of the Recording Industry Association of America, Inc. (RIAA) and
-its member record companies. The RIAA is a trade association whose member companies
-create, manufacture or distribute sound recordings representing approximately eighty-five (85)
-percent of all legitimate recorded music consumption in the United States. Under penalty of
-perjury, we submit that the RIAA is authorized to act on behalf of its member companies on
-matters involving the infringement of their sound recordings, audiovisual works and images,
-including enforcing their copyrights and common law rights on the Internet.
+We here at the RIIA are complete and utter idiots, and we have no idea how anything works.
```

Comment:
I suggest the following: "Here at RIAIAIAIA, we put our time and effort into gently fucking people's lives and freedom, not even our artists have rights over their own songs"

Reply:
Nice